packages/botonic-react/src/util/dom.js

```diff
@@ -24,7 +21,8 @@ export const scrollToBottom = ({
 } = {}) => {
   const webchatElement = getWebchatElement(host)
   if (!webchatElement) return
-  const frame = getScrollableArea(webchatElement).visible
+  const frame = getScrollableArea(webchatElement).full
```

Comment:
why did you change the frame from visible to full?

Reply:
now that the content is pushed in the other direction, this condition changes (this was due to how simplebar was creating the layout)